cdap-security/src/main/java/io/cdap/cdap/security/auth/ProxyUserIdentityExtractor.java

```diff
@@ -64,8 +64,8 @@ public UserIdentityExtractionResponse extract(HttpRequest request) throws UserId
                                                 "No user identity found");
     }
 
-    UserIdentity userIdentityObj = new UserIdentity(userIdentity, new LinkedHashSet<>(),
-                                                    now, now + EXPIRATION_SECS);
+    UserIdentity identifier = new UserIdentity(userIdentity, UserIdentity.IdentifierType.EXTERNAL,
```

Comment:
why `identifier`? What does it identify?

Reply:
Was part of a reverted refactor; went ahead and reverted this change.

Reply:
I don't see the revert